base/deprecated.jl

```diff
@@ -80,7 +80,7 @@ function firstcaller(bt::Array{Ptr{Void},1}, funcsym::Symbol)
         if lkup === ()
             continue
         end
-        fname, file, line, fromC = lkup
+        fname, file, line, inlined_file, inlined_line, fromC = lkup
```

Comment:
They mean "where the code was inlined from".

> It seems potentially annoying to deal with extra information about inlining for each source location. I would expect inlined frames to be represented the same as normal ones, perhaps with an inline flag set to true or something.

Each address in the backtrace may have inlining information associated with it; there is no separate frame ip for inlined code. Returning the inlining info here seemed simplest option to fit in the current implementation of `jl_getFunctionInfo`/`lookup_code_address`.

`jl_getFunctionInfo` may eventually need a redesign to return a chain of inline information (for nested code), but this PR is not emitting that (intermediates) debug info either.